addons/packages/harbor/2.2.3/test/go.sum

```diff
@@ -1,10 +1,14 @@
+github.com/creack/pty v1.1.9 h1:uDmaGzcdjhF4i/plgjmEsriH11Y0o7RKapEf/LDaM3w=
```

Comment:
@stmcginnis can we revert go.sum modifications to this package. While I'm sure they're correct, we should prevent this tree from being mutated by this PR.